extensions/reactive-datasource/runtime/src/main/java/io/quarkus/reactive/datasource/runtime/ReactiveDatasourceHealthCheck.java

```diff
@@ -64,7 +64,7 @@ public HealthCheckResponse call() {
                                 });
                     });
                 } else {
-                    log.debug("Vert.x context not found for health check");
+                    log.warn("Vert.x context unavailable to perform healthcheck of reactive datasource `" + dataSourceName + "`. This is unlikely to work correctly.");
```

Comment:
```
This is unlikely to work correctly.
```

Are you sure about this? I think it should be possible to run this without a context. Not sure why we had that issue only for MySQL and only on CI.

Reply:
You'll have the previous behaviour, which seems to timeout frequently. I'll keep the warning for now, as running w/o a Vert.x context seems tricky for other reasons as well.